app/views/admin/claims/show.html.erb

```diff
@@ -32,6 +38,32 @@
       </div>
     <% end %>
 
+    <% if @claim_ids_with_matching_attributes.present? %>
+      <table id="claims-with-matches" class="govuk-table govuk-!-margin-bottom-9">
+        <caption class="govuk-table__caption govuk-heading-l">Claims with matching details</caption>
+        <thead class="govuk-table__head">
+          <tr class="govuk-table__row">
+            <th scope="col" class="govuk-table__header">Claim</th>
+            <th scope="col" class="govuk-table__header">Matching details</th>
+          </tr>
+        </thead>
+        <tbody class="govuk-table__body">
+          <% @claim_ids_with_matching_attributes.each do |claim_id, matching_attributes| %>
+            <tr class="govuk-table__row">
+              <th scope="row" class="govuk-table__header"><%= link_to Claim.find(claim_id).reference, admin_claim_path(Claim.find(claim_id)), class: "govuk-link" %>
```

Comment:
For each matching claim, this makes a query to load the claim so that we can display its reference. But, when forming the list of matching claims, inside `Claim::MatchingAttributeFinder`, we already loaded the list of all the claims, and then discarded them, just returning the claims' IDs. Maybe we could change the interface of that class so that it returns the claim objects instead of / in addition to their IDs?

A separate point that will hopefully become moot: this loads each claim _twice_; once for the reference and another time for the URL.